demos/sock-shop/sock-shop-loadgen.yaml

```diff
@@ -28,10 +28,10 @@ spec:
 
       initContainers:
       - name: wait-sock-shop
-        image: alpine:3.6
+        image: gcr.io/pixie-oss/pixie-dev-public/curl:1.0
```

Comment:
can you use the multiarch version instead: `gcr.io/pixie-oss/pixie-dev-public/curl:multiarch-7.87.0`

Reply:
Where is the dockerfile for that one?

Reply:
It's a meta image that points to the same dockerfile, but with arm and x86 versions that are picked based on the architecture of the docker platform

Reply:
Oh wait actually, we used the official `curlimages/curl` for this multiarch image